direct/data/mri_transforms.py

```diff
@@ -346,27 +453,31 @@ def __call__(
             Coil dimension. Default: 0.
         spatial_dims: (int, int)
             Spatial dimensions corresponding to (height, width). Default: (1, 2).
+        complex_dim: int
+            Complex dimension. Used if `type_reconstruction` is either "complex_mod" or "sense_mod" Default: -1.
 
         Returns
         ----------
         sample: dict
-            Contains key target_key with value a torch.Tensor of shape (*spatial_dims) or (*spatial_dims) if
-            type_reconstruction is 'rss'.
+            Contains key target_key with value a torch.Tensor of shape (*spatial_dims) if `type_reconstruction` is
+            "rss", "complex_mod" or "sense_mod", and of shape(*spatial_dims, complex_dim=2) otherwise.
         """
         kspace_data = sample[self.kspace_key]
 
         # Get complex-valued data solution
         image = self.backward_operator(kspace_data, dim=spatial_dims)
-        if self.type_reconstruction == "complex":
+        if self.type_reconstruction in ["complex", "complex_mod"]:
             sample[self.target_key] = image.sum(coil_dim)
-        elif self.type_reconstruction.lower() == "rss":
+        elif self.type_reconstruction == "rss":
             sample[self.target_key] = T.root_sum_of_squares(image, dim=coil_dim)
-        elif self.type_reconstruction == "sense":
+        elif self.type_reconstruction in ["sense", "sense_mod"]:
             if "sensitivity_map" not in sample:
                 raise ValueError("Sensitivity map is required for SENSE reconstruction.")
             sample[self.target_key] = T.complex_multiplication(T.conjugate(sample["sensitivity_map"]), image).sum(
                 coil_dim
             )
+        if "mod" in self.type_reconstruction:
+            sample[self.target_key] = T.modulus(sample[self.target_key], complex_dim)
```

Comment:
Enumerate the possibilities, don't do string matching

Reply:
Done